src/interactions/panZoomInteraction.ts

```diff
@@ -232,6 +255,92 @@ namespace Plottable.Interactions {
       return compareF(zoomAmount, boundingDomainExtent / domainExtent);
     }
 
+    private _constrainZoomValues(scale: QuantitativeScale<number>, zoomAmount: number, centerPoint: number) {
```

Comment:
I didn't think through the math here but just want to make sure this will work for logarithmic scales as well?